apidef/oas/linter_test.go

```diff
@@ -83,6 +83,7 @@ func TestXTykGateway_Lint(t *testing.T) {
 		}
 
 		settings.Upstream.RateLimit.Per = ReadableDuration(10 * time.Second)
+		settings.Server.Authentication.KeyRetentionPeriod.Value = settings.Upstream.RateLimit.Per
```

Comment:
```suggestion
		settings.Server.Authentication.KeyRetentionPeriod.Value = ReadableDuration(10 * Second)
```
I see what you did and I don't like it :D (cognitive load / indirection, even if just N-1 line)